src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/records/LinearNonLocatableCopyRatio.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.records;
+
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+
+/**
+ * Represents a value of the denoised copy ratio output by {@link GermlineCNVCaller}
+ *
+ * @author Andrey Smirnov &lt;asmirnov@broadinstitute.org&gt;
+ */
+public class LinearNonLocatableCopyRatio {
+    private final double denoisedCopyRatio;
+
+    public LinearNonLocatableCopyRatio(final double denoisedCopyRatio) {
+        this.denoisedCopyRatio = denoisedCopyRatio;
+    }
+
+    public double getDenoisedCopyRatio() {
+        return denoisedCopyRatio;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Add some braces, make `that` final, and clean up this automatically generated code.

Reply:
Done